### PR TITLE
feat #25: cache client support multiple key test and fix log

### PR DESCRIPTION
**Pull request by shibd**
#25

### CODE REVIEW

File: chaos-framework/src/main/java/io/openchaos/model/CacheModel.java [hunk 1]
```diff
@@ -101,14 +101,14 @@ private static CacheChaosDriver createCacheChaosDriver(File driverConfigFile) th
             log.info("Cache clients setup..");
 
             for (int i = 0; i < concurrency; i++) {
-                Client client = new CacheClient(cacheChaosDriver, recorder, key);
+                Client client = new CacheClient(cacheChaosDriver, recorder, Optional.of(key.get() + i));
```
Comment (shibd) [inline chaos-framework/src/main/java/io/openchaos/model/CacheModel.java:104]: Each client sends a different key

File: chaos-framework/src/main/java/io/openchaos/model/CacheModel.java [hunk 2]
```diff
@@ -185,15 +186,17 @@ private static CacheChaosDriver createCacheChaosDriver(File driverConfigFile) th
...
+        log.info("Cache chaos test stop");
         workers.forEach(Worker::breakLoop);
     }
 
     @Override public void afterStop() {
         if (clients.isEmpty()) {
             throw new IllegalArgumentException("clients is empty");
         } else {
-            clients.get(0).lastInvoke();
+            for (Client client : clients) {
```
Comment (shibd) [inline chaos-framework/src/main/java/io/openchaos/model/CacheModel.java:197]: sum all key data

**Comment by RongtongJin** [comment]
Good catch! Could you solve the conflict first?

**Comment by shibd** [comment]
I'll fix it

**Comment by shibd** [comment]
Can you help me review? @RongtongJin @vongosling

File: chaos-framework/src/main/java/io/openchaos/model/KVModel.java [hunk 3]
```diff
@@ -146,14 +146,15 @@ private static KVDriver createCacheChaosDriver(File driverConfigFile) throws IOE
             }
 
             log.info("Cluster start...");
-            log.info("Wait for all nodes to start...");
+            log.info("Wait for all preNodes to start...");
```
Comment (vongosling) [inline chaos-framework/src/main/java/io/openchaos/model/KVModel.java:149]: preNode -> metaNode

**Pull request closed without merging**